src/Form/Type/OrderDiscountCollectionType.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusOrderEditPlugin\Form\Type;
+
+use Sylius\Component\Core\Model\Adjustment;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Order\Model\AdjustmentInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class OrderDiscountCollectionType extends AbstractType
+{
+    private const CUSTOM_ORDER_DISCOUNT = 'custom_order_discount';
+
+    public function getParent()
+    {
+        return CollectionType::class;
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'entry_type' => OrderDiscountType::class,
+            'allow_add' => true,
+            'allow_delete' => true,
+            'by_reference' => false,
+            'label' => 'sylius.form.order.discounts',
+            'entry_options' => [
+                'label' => false,
+                'currency' => 'USD',
+            ],
+            'getter' => function (OrderInterface &$order): array {
+                $adjustments = $order->getAdjustments(self::CUSTOM_ORDER_DISCOUNT)->toArray();
```

Comment:
I usually namespace adjustment types if I create them, and I usually place them in an `AdjustmentInterface` just to be consistent with Sylius. The type could be `setono_admin_order_discount` or somethiing like that

Reply:
To be honest, placing constants in interfaces was not the best idea in Sylius :dancer: Interfaces are meant to be contracts describing the behavior of classes that implement them, not the handy containers for strings to not have them hardcoded - so such usage is principally wrong (even though I agree would be consistent with Sylius 🚀). How about having a final class with a private constructor named `OrderEditDiscountTypes`? I believe we would use such an approach if we think about it earlier in Sylius xD

Reply:
Could we call it `AdjustmentTypes` then? Just so that if people search for it in PHPStorm it comes up if they with `Adjustment...`

Reply:
Can be 👍